<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">

<head>

  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
  
  <title>Bio 414 Lecture Notes 13</title>
<link href="/theme/css/style.css" rel="stylesheet" type="text/css" media="screen" />
<link href="/" type="application/atom+xml" rel="alternate" title="The Commonplace Index ATOM Feed" />
  
  <!--[if IE]>
  <style type="text/css"> 
  .twoColFixRtHdr #mainContent { zoom: 1; }
  </style>
  <![endif]-->

</head>

<body>

<div id="container">

  <div id="header">
  <h1><a href="/index.html">The Commonplace Index</h1>
  
  </div><!-- end #header -->
  
  <div id="topMenu">
  
    <ul>
    	<li><a href="/index.html">Home</a></li>
        <li>&ndash;</li>
        <li><a href="/archives.html">Archives</a></li>
        
    </ul>
  
  </div><!-- end #topMenu -->

  <div id="sidebar">
    <ul>
	<li>
	    <h3>Pages</h3>
	    <ul>
	    </ul>
	</li>
	<li>
            <h3>Categories</h3>
            <ul>
		     <li ><a href="/category/blog.html">Blog</a></li>
		     <li class="active"><a href="/category/lecture-notes.html">Lecture Notes</a></li>
	    </ul>
	</li>
	<li>
		<h3>Blogroll</h3>
		<ul>
                    <li><a href="http://getpelican.com/">Pelican</a></li>
                    <li><a href="http://python.org/">Python.org</a></li>
                    <li><a href="http://jinja.pocoo.org/">Jinja2</a></li>
		</ul>
	</li>
        <li>
                <h3>Social</h3>
                <ul>
                    <li><a href="#">Maybe coming soon</a></li>
                </ul>
        </li><!-- /.social -->
    </ul>
  </div><!-- end #sidebar -->

  <div id="mainContent">
  	
 


<div class="blogItem">
    
      <h2><a href="/bio-414-lecture-notes-13.html">Bio 414 Lecture Notes 13</a></h2>
      <h3>Tue 17 February 2015</h3>
      
  <p>20150217_140000_bio_414_lecture_notes_13</p>
<h1>Notes</h1>
<h2>Rate Heterogeneity</h2>
<h3>Models with Rate Heterogeneity</h3>
<p>There are sites where mutations do arise, but some sites become fixed and some don't. That is to say, different sites will tend to have different evolution rates, especially if they are far apart (think of recombination distance). </p>
<table>
<thead>
<tr>
<th align="center"><span class="math">\(r_{1}\)</span></th>
<th align="center"><span class="math">\(r_{2}\)</span></th>
<th align="center">...</th>
<th align="center"><span class="math">\(r_{n}\)</span></th>
</tr>
</thead>
<tbody>
<tr>
<td align="center">A</td>
<td align="center">C</td>
<td align="center">...</td>
<td align="center">G</td>
</tr>
<tr>
<td align="center">A</td>
<td align="center">C</td>
<td align="center">...</td>
<td align="center">A</td>
</tr>
<tr>
<td align="center">G</td>
<td align="center">C</td>
<td align="center">...</td>
<td align="center">G</td>
</tr>
</tbody>
</table>
<p>Q matrix (above) is multiplied by <span class="math">\(r_{i}\)</span> for each site <span class="math">\(i\)</span>. </p>
<p>Gamma (<span class="math">\(\alpha,\alpha\)</span>) distribution shows different rates.</p>
<ul>
<li>Small <span class="math">\(\alpha\)</span> (e.g. <span class="math">\(\alpha&lt;1\)</span>): High level of rate variation. </li>
<li>Large <span class="math">\(\alpha\)</span> (e.g. <span class="math">\(\alpha&gt;1\)</span>): Low level of rate variation. </li>
</ul>
<p>In practice, we can't estimate for all <span class="math">\(r_{n}\)</span>,but we can estimate <span class="math">\(\alpha\)</span>, it's easier to say what the sites are doing on average. We can do this by binning all of the <span class="math">\(r\)</span> into four bins (though we can increase or decrease this number, four is just the typically used one), this helps with computational efficiency. We can also model proportion of invariant sites (kind of opposite of rate heterogeneity, accounts for sites that can never vary), e.g. F84+I+G (F84 model with invariant sites and gamma rates).  </p>
<h2>Test Review</h2>
<h3>Basic Genetics</h3>
<p><strong>Central dogma:</strong> Organization of genetic information, <span class="math">\(DNA \rightarrow RNA \rightarrow Protein\)</span>.</p>
<p><strong>Sequencing Technologies:</strong> The automation and parallelization of individual steps that gives us the sequence of an organism. </p>
<p><strong>Genetic Terminology:</strong> </p>
<ul>
<li>Locus(loci) </li>
<li>Markers </li>
<li>Allele </li>
<li>Polymorphic loci</li>
<li>Types of mutation <ul>
<li>Point mutations</li>
<li>Transitions</li>
<li>Transversions</li>
</ul>
</li>
<li>Indels</li>
<li>Microsatellites. </li>
</ul>
<p><strong>Genetic drift and selection:</strong> Neutral mutations can fix in population due to drift. Selection changes the probability of fixation, which allows non-neutral mutations to fit more/ less frequently. </p>
<h3>Sequence Alignment</h3>
<p><strong>Sequence alignment:</strong> Align homologous nucleotides or amino acids.</p>
<ul>
<li><strong>Sequence alignment score:</strong> <span class="math">\(\sum S(a,b) - \sum W(l)\)</span>. First part is summing over all aligned pairs, where <span class="math">\(S\)</span> is elements of the scoring matrix. The second part sum over all gaps, with <span class="math">\(l\)</span> specifying gap length. Gap penalties are as follow:<ul>
<li><strong>Linear:</strong> <span class="math">\(W(l) = g*l\)</span>. </li>
<li><strong>Affine:</strong> <span class="math">\(W(l) = g_{open} + g_{extension} (l-1)\)</span></li>
</ul>
</li>
</ul>
<p><strong>Alignment Algorithms:</strong></p>
<ul>
<li><strong>Needleman-Wunsch Algorithm:</strong> Global alignment. </li>
<li><strong>Smith-Waterson algorithm:</strong> Local alignment.</li>
</ul>
<p><strong>BLAST:</strong> Stands for Basic Local Alignment Search Tool. BLAST is use to find a match for our query sequence in a database. </p>
<p><strong>E-value:</strong> Expected # of database sequence matches to our query sequence, with a score of <span class="math">\(\geq S^{*}\)</span>. </p>
<p>Get picture. </p>
<ul>
<li>We would simulate random sequence that we can run through BLAST.</li>
<li>If <span class="math">\(S^{*}\)</span> is a score of a "good" alignment, then numbers (<span class="math">\(n_{1},...n_{1000}\)</span>) should be low (mostly zeros). </li>
</ul>
<h3>Evolutionary Distance</h3>
<p><strong>Hamming Distance:</strong> A proportion of polymorphic sites.</p>
<div class="math">$$D = \frac{\#\ of\ polymorphic\ sites}{\#\ of\ total\ sites}$$</div>
<p><strong>Jukes-Cantor Distance:</strong></p>
<div class="math">$$\hat{d} = - \frac{3}{4} \ln (1-\frac{4}{3}D)$$</div>
<p>Get the side notes. </p>
<p><strong>Subsitution models:</strong></p>
<ul>
<li>What is needed for the exam is to be able to understand nested relationships among the models. </li>
<li>E.g. If we artificially make <span class="math">\(\pi_{A} = \pi_{G} = \pi_{C} = \pi_{T}\)</span> in HKY model, it becomes K2P model. </li>
</ul>
<h3>Phylogenetic Trees</h3>
<p><strong>Terms:</strong></p>
<ul>
<li>Nodes (internal &amp; tips). </li>
<li>Branches/ edges.</li>
<li>Branch length.</li>
<li>Newick format.</li>
<li>Cladogram. </li>
<li>Additive tree.</li>
<li>Ultrametric tree. </li>
<li>Rooted tree. </li>
<li>Unrooted trees. </li>
</ul>
<p>You should be able to use root a tree (get picture from lecture). </p>
<p>Be able to recognize splits. E.g. in picture above {1,2,3}, {4,5,6,7} is a split in the tree on the left, whereas {1,2,6}, {3,4,5,7}.</p>
<h3>Distance-Based Phylogenetic Reconstruction</h3>
<p><strong>Least-square Method:</strong> Returns tree with branch length that minimizes:</p>
<div class="math">$$\sum_{i&lt;j}(d_{ij}-e_{ij})^{2}$$</div>
<p>Where <span class="math">\(d_{ij}\)</span> is the sequence distance and <span class="math">\(e_{ij}\)</span> is the tree distance.</p>
<p><strong>Minimum Evolution Method:</strong> Uses least square to estimate a branch length for each tree, returns a tree with the minimum total (sum of all branch) length. </p>
<p><strong>UPGMA:</strong>  An approximation (a faster way that returns close enough results) of least-square but with constraint of ultrametric trees (molecular clock). </p>
<p><strong>NJ:</strong> An approximation (same as above) of minimum evolution with no constraints on branch length. </p>
<p><strong>Maximum Parsimony:</strong> Returns a phylogeny that has the minimum number of needed substitutions to explain the data given. </p>
<p>Problems: </p>
<ul>
<li>Can't attach parameters (it's hard to extend), or use it to estimate parameters. </li>
<li>It's statistically inconsistent. The more data you attached, the less accurate the tree will be. </li>
<li>Long-branch attraction, resulting from not being able to attach parameters (like branch length). It's statistically inconsistent. </li>
</ul>
<p><strong>Maximum Likelihood:</strong> Returns a combination of both the phylogeny and the parameters that made up the model that estimated the phylogeny. Maximize the probability of observing the given data. </p>
<p>Problems: </p>
<ul>
<li>Computationally costly. </li>
<li>Model needs to be correctly specified (and we can never be completely sure of our parameters for the model we give ML). </li>
</ul>
<script type="text/javascript">if (!document.getElementById('mathjaxscript_pelican_#%@#$@#')) {
    var align = "center",
        indent = "0em",
        linebreak = "false";

    if (false) {
        align = (screen.width < 768) ? "left" : align;
        indent = (screen.width < 768) ? "0em" : indent;
        linebreak = (screen.width < 768) ? 'true' : linebreak;
    }
    
    var mathjaxscript = document.createElement('script');
    mathjaxscript.id = 'mathjaxscript_pelican_#%@#$@#';
    mathjaxscript.type = 'text/javascript';
    mathjaxscript.src = '//cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML';
    mathjaxscript[(window.opera ? "innerHTML" : "text")] =
        "MathJax.Hub.Config({" +
        "    config: ['MMLorHTML.js']," +
        "    TeX: { extensions: ['AMSmath.js','AMSsymbols.js','noErrors.js','noUndefined.js'], equationNumbers: { autoNumber: 'AMS' } }," +
        "    jax: ['input/TeX','input/MathML','output/HTML-CSS']," +
        "    extensions: ['tex2jax.js','mml2jax.js','MathMenu.js','MathZoom.js']," +
        "    displayAlign: '"+ align +"'," +
        "    displayIndent: '"+ indent +"'," +
        "    showMathMenu: true," +
        "    tex2jax: { " +
        "        inlineMath: [ ['\\\\(','\\\\)'] ], " +
        "        displayMath: [ ['$$','$$'] ]," +
        "        processEscapes: true," +
        "        preview: 'TeX'," +
        "    }, " +
        "    'HTML-CSS': { " +
        "        styles: { '.MathJax_Display, .MathJax .mo, .MathJax .mi, .MathJax .mn': {color: 'inherit ! important'} }," +
        "        linebreaks: { automatic: "+ linebreak +", width: '90% container' }," +
        "    }, " +
        "}); ";
    (document.body || document.getElementsByTagName('head')[0]).appendChild(mathjaxscript);
}
</script>	
      
      <h3 class="blogMeta">By <a href="#">Cao-chau Vu</a>, Category: <a href="/category/lecture-notes.html">Lecture Notes</a></h3>
    
    </div><!-- end #blogItem -->
				
				<!-- end #content -->
  
  </div><!-- end #mainContent -->

<div class="clearfloat"></div>

  <div id="footer">
  
    <p><a href="http://css4free.com/" title="free CSS web site designs">Free CSS Gallery</a></p>
    <p>Proudly powered by <a href="http://getpelican.com">pelican</a>, which takes great advantages of <a href="http://python.org">python</a>.
  </div><!-- end #footer -->

</div><!-- end #container -->
</body>
</html>