<!DOCTYPE html>
<html lang="en">
<head>
        <title>Bio 414 Lecture Notes 04</title>
        <meta charset="utf-8" />
        <link rel="stylesheet" href="/theme/css/main.css" type="text/css" />
   

        <!--[if IE]>
                <script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->

        <!--[if lte IE 7]>
                <link rel="stylesheet" type="text/css" media="all" href="/css/ie.css"/>
                <script src="/js/IE8.js" type="text/javascript"></script><![endif]-->

        <!--[if lt IE 7]>
                <link rel="stylesheet" type="text/css" media="all" href="/css/ie6.css"/><![endif]-->

</head>

<body id="index" class="home">
        <header id="banner" class="body">
                <h1><a href="">Commonplace Book </a></h1>
                <nav><ul>
                    <li ><a href="/category/blog.html">Blog</a></li>
                    <li class="active"><a href="/category/lecture-notes.html">Lecture Notes</a></li>
                </ul></nav>
        </header><!-- /#banner -->
<section id="content" class="body">
  <article>
    <header>
      <h1 class="entry-title">
        <a href="bio-414-lecture-notes-04.html" rel="bookmark"
           title="Permalink to Bio 414 Lecture Notes 04">Bio 414 Lecture Notes 04</a></h1>
    </header>

    <div class="entry-content">
<footer class="post-info">
        <abbr class="published" title="2015-01-15T00:00:00+01:00">
                Thu 15 January 2015
        </abbr>

        <address class="vcard author">
                By <a class="url fn" href="/author/cao-chau-vu.html">Cao-chau Vu</a>
        </address>
<p>In <a href="/category/lecture-notes.html">Lecture Notes</a>. </p>
</footer><!-- /.post-info -->      <p>20150115_140000_bio_414_lecture_notes</p>
<h1>Notes</h1>
<h2>Sequence Alignments</h2>
<ul>
<li>Used in all evolutionary analysis</li>
<li>Goal of sequence alignment is to group homologous nucleotides/ aa that most likely share a common ancestry.</li>
</ul>
<h2>Good vs. Bad Alignments:</h2>
<p>Take for example the two following sequence</p>
<blockquote>
<p>Alignment 1</p>
<p>CAGT_ATTT</p>
<p>CAGTCAT__</p>
<p>Alignment 2</p>
<p>_CAGTATTT</p>
<p>CA_GTC_AT</p>
</blockquote>
<ul>
<li>The first alignment is good whereas the second is bad.<ul>
<li>The # of gaps is the same between the two.</li>
<li>However the first one has more consecutive matches.</li>
</ul>
</li>
</ul>
<blockquote>
<p>Alignment 3</p>
<p>CAGTATTT</p>
<p>CAGTC_AT</p>
</blockquote>
<ul>
<li>Alignment 3 could be either.<ul>
<li>Less gaps</li>
<li>But also less consecutive matches.</li>
</ul>
</li>
<li>So we need a way to formalize what is a good and bad match, in order to have a computer automate it.</li>
</ul>
<h2>Scoring System</h2>
<p>Has two parts.</p>
<ol>
<li>Scoring matrix:<ul>
<li>With elements S(a,b)</li>
<li>a and b are two nucleotides/ aa. </li>
<li>For nucleotides a simple matrix works well.  </li>
<li>E.g. A 1 if the nucleotides match, 0 if they don't. </li>
<li>AA have more complicated matrices.</li>
</ul>
</li>
<li>Gap penalty function: <ul>
<li>W(l)</li>
<li>Penalty of a gap of length L.</li>
<li>Simplest gap penalty function is linear: W(l) = gl</li>
<li>g is the penalty for a gap of length 1.</li>
<li>When doing parallel sequencing, there is no gap in the same position of both alignment because we're comparing indels. The sequences have to be parallel to each other so one insertion is another's deletion.</li>
</ul>
</li>
</ol>
<p>What happens if we use the g = 1 to score the sequences above </p>
<blockquote>
<p>Alignment 1</p>
<p>CAGT_ATTT</p>
<p>CAGTCAT__</p>
</blockquote>
<p>We have 6 matches, and 3 gaps, so resulting score is 6-3 = 3.</p>
<blockquote>
<p>Alignment 2</p>
<p>_CAGTATTT</p>
<p>CA_GTC_AT</p>
</blockquote>
<p>We have 3 matches, 3 gaps and 3 mismatches. The resulting score is 3-3+0 = 0.</p>
<p>3&gt;0, so alignment 1 is better than alignment 2. Now what about alignment 3?</p>
<blockquote>
<p>Alignment 3</p>
<p>CAGTATTT</p>
<p>CAGTC_AT</p>
</blockquote>
<p>We have 5 matches, 2 mismatches and 1 gap, so 5-1+0 = 4. 4&gt;3 so alignment 3 is better than alignment 1 by our algorithm.</p>
<p>Sequence alignment score is the sum of all aligned gaps subtracted from the sum of all aligned pairs:</p>
<p><strong>Sequence alignment score</strong> = $\sum S(a,b) - \sum w(l)$</p>
<ul>
<li>Linear gap penalty function however is unrealistic, because gaps next to each other are not independent (for example slippage event due to RNA polymerase skipping multiple bases at once).</li>
<li>A modification would be to penalize one large gaps less than multiple small gaps of the same total length.</li>
</ul>
<p>[w(l) = g_open + g_ext (l-1)]</p>
<p>where $g_{ext} &lt; g_{open}$</p>
<ul>
<li>$g_{open}$ = gap opening penalty, which is the penalty for being the first letter of a gap.</li>
<li>$g_{ext}$ = gap extension penalty, which is a smaller penalty for lengths larger than 1.</li>
</ul>
<h2>Alignment Scoring</h2>
<p>Check out both the bolded terms immediately below</p>
<ul>
<li><strong>Needlman-Wunsch Algorithm</strong><ul>
<li>Objective: Find best <em>global</em> alignment of 2 sequences.</li>
<li>Use <strong>dynamic programming</strong>.</li>
<li>Matrix method.</li>
<li>Filled from upper left to lower right, reversed if you're backtracking.</li>
<li>Diagonal arrows correspond to aligned pairs.</li>
<li>Horizontal/ vertical arrows correspond to gaps.</li>
</ul>
</li>
<li><strong>Smith-Waterman Algorithm:</strong><ul>
<li>Objective: Find best <em>local</em> alignment  </li>
<li>Only some parts of the two sequence aligned.</li>
<li>Good for aligning short sequence to long sequence.</li>
<li>Use scenarios:</li>
<li>Genome assembly in NGS read alignment.</li>
<li>Database search, asking "what have you sequenced?". <ul>
<li>Objective: Find a match to the query sequence in a database.</li>
<li>Usually done in BLAST in GeneBank, but could be other database.</li>
<li>BLAST = Basic Local Alignment Search Tool.</li>
</ul>
</li>
</ul>
</li>
</ul>
    </div><!-- /.entry-content -->

  </article>
</section>
        <section id="extras" class="body">
                <div class="blogroll">
                        <h2>blogroll</h2>
                        <ul>
                            <li><a href="http://getpelican.com/">Pelican</a></li>
                            <li><a href="http://python.org/">Python.org</a></li>
                            <li><a href="http://jinja.pocoo.org/">Jinja2</a></li>
                            <li><a href="#">You can modify those links in your config file</a></li>
                        </ul>
                </div><!-- /.blogroll -->
                <div class="social">
                        <h2>social</h2>
                        <ul>
                            
                            <li><a href="#">You can add links in your config file</a></li>
                            <li><a href="#">Another social link</a></li>
                        </ul>
                </div><!-- /.social -->
        </section><!-- /#extras -->

        <footer id="contentinfo" class="body">

        </footer><!-- /#contentinfo -->

</body>
</html>