<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="chrome=1">
    <title>The Commonplace Index</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" href="/theme/styles.css">
    <script src="//ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
    <link rel="stylesheet" href="/theme/slicknav.css">
    <script src="/theme/jquery.slicknav.min.js"></script>
    <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
    <script type="text/javascript"
	    src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
    </script>
    <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.2/html5shiv.js"></script>
    <![endif]-->
<script>
	$(function(){
		$('#menu').slicknav({'label':''});
	});
</script>
<link rel="icon" href="/theme/favicon.ico" sizes="16x16 32x32 48x48 64x64" type="image/vnd.microsoft.icon">
<link rel="apple-touch-icon" sizes="114x114" href="/theme/apple-touch-icon-precomposed.png">
<link rel="apple-touch-icon" sizes="152x152" href="/theme/favicon152.png">
<link rel="apple-touch-icon" sizes="196x196" href="/theme/favicon196.png">
  </head>

  <body>
    <div id="wrapper">

      <header>
	<nav class="byline"><ul id="menu">
        </ul></nav><!-- /#menu -->
        <div class="downloads">
          <a href="#" class="fa fa-Maybe coming soon"></a>
        </div>
        <hgroup>
          <h1>The Commonplace Index</h1>
          <h2 class="tagline"></h2>
        </hgroup>
      </header>
<section id="content">
<ul id="post-list">
        <li><article class="hentry">
            <header class="article-list"> <h2 class="entry-title"><a href="/bio-414-lecture-notes-13.html" rel="bookmark" title="link to Bio 414 Lecture Notes 13">Bio 414 Lecture Notes 13</a></h2>
                <footer class="post-info">
                    <address class="vcard author">By
                        <a class="url fn" href="/author/cao-chau-vu.html">Cao-chau Vu</a>
                    </address>
                    <abbr class="published" title="2015-02-17T00:00:00+01:00">  on 17.02.2015 </abbr>
                </footer><!-- /.post-info -->
	    </header>
                <div class="entry-content"> <p>20150217_140000_bio_414_lecture_notes_13</p>
<h1>Notes</h1>
<p>All images below are taken from a Biology 414 lecture by Dr. Vladimir Minin at University of Washington. With notes taken from past lecture notes added to supplement review. </p>
<h2>Rate Heterogeneity</h2>
<h3>Models with Rate Heterogeneity</h3>
<p>There are sites where mutations do arise, but some sites become fixed and ...</p> </div><!-- /.entry-content -->
        </article></li>
        <li><article class="hentry">
            <header class="article-list"> <h2 class="entry-title"><a href="/bio-414-lecture-notes-12.html" rel="bookmark" title="link to Bio 414 Lecture Notes 12">Bio 414 Lecture Notes 12</a></h2>
                <footer class="post-info">
                    <address class="vcard author">By
                        <a class="url fn" href="/author/cao-chau-vu.html">Cao-chau Vu</a>
                    </address>
                    <abbr class="published" title="2015-02-12T00:00:00+01:00">  on 12.02.2015 </abbr>
                </footer><!-- /.post-info -->
	    </header>
                <div class="entry-content"> <p>20150212_140000_bio_414_lecture_notes </p>
<h1>Notes</h1>
<h2>MLE Review</h2>
<p>The MLE method find parameter(s) that maximize the probability of observed data. </p>
<p>Let's say we fix the phylogeny and keep only transition/ transversion ratio as free parameter (Kimura model), the maximum likelihood estimation will kind of look like a parabola. Where the maximum point ...</p> </div><!-- /.entry-content -->
        </article></li>
        <li><article class="hentry">
            <header class="article-list"> <h2 class="entry-title"><a href="/bio-414-lecture-notes-11.html" rel="bookmark" title="link to Bio 414 Lecture Notes 11">Bio 414 Lecture Notes 11</a></h2>
                <footer class="post-info">
                    <address class="vcard author">By
                        <a class="url fn" href="/author/cao-chau-vu.html">Cao-chau Vu</a>
                    </address>
                    <abbr class="published" title="2015-02-10T00:00:00+01:00">  on 10.02.2015 </abbr>
                </footer><!-- /.post-info -->
	    </header>
                <div class="entry-content"> <p>20150210_140000_bio_414_lecture_notes_11</p>
<h1>Notes</h1>
<p>Today is all about <strong>maximum likelihood phylogenetics</strong>. </p>
<h2>Maximum Likelihood Phylogenetics Examples</h2>
<h3>A 2 Taxa Case</h3>
<p>If we have one ancestor (X, for having unknown base X) with two descendants (A,G) with corresponding time (<span class="math">\(t_{1}, t_{2}\)</span>), that is: </p>
<div class="math">$$(A: t_{1},G:t_{2})X$$</div>
<p>The ...</p> </div><!-- /.entry-content -->
        </article></li>
        <li><article class="hentry">
            <header class="article-list"> <h2 class="entry-title"><a href="/bio-414-lecture-notes-10.html" rel="bookmark" title="link to Bio 414 Lecture Notes 10">Bio 414 Lecture Notes 10</a></h2>
                <footer class="post-info">
                    <address class="vcard author">By
                        <a class="url fn" href="/author/cao-chau-vu.html">Cao-chau Vu</a>
                    </address>
                    <abbr class="published" title="2015-02-05T00:00:00+01:00">  on 05.02.2015 </abbr>
                </footer><!-- /.post-info -->
	    </header>
                <div class="entry-content"> <p>20150205_140000_bio_414_lecture_notes </p>
<h1>Notes</h1>
<h2>Parsimony Continued</h2>
<h3>Generalized Parsimony</h3>
<p>We added 1 for every substitution required to explain site data, this corresponds to a (step) matrix like: </p>
<table>
<thead>
<tr>
<th></th>
<th>A</th>
<th>G</th>
<th>C</th>
<th>T</th>
</tr>
</thead>
<tbody>
<tr>
<td>A</td>
<td>0</td>
<td>1</td>
<td>1</td>
<td>1</td>
</tr>
<tr>
<td>G</td>
<td>1</td>
<td>0</td>
<td>1</td>
<td>1</td>
</tr>
<tr>
<td>C</td>
<td>1</td>
<td>1</td>
<td>0</td>
<td>1</td>
</tr>
<tr>
<td>T</td>
<td>1</td>
<td>1</td>
<td>1</td>
<td>0</td>
</tr>
</tbody>
</table>
<p>What if ...</p> </div><!-- /.entry-content -->
        </article></li>
        <li><article class="hentry">
            <header class="article-list"> <h2 class="entry-title"><a href="/bio-414-lecture-notes-09.html" rel="bookmark" title="link to Bio 414 Lecture Notes 09">Bio 414 Lecture Notes 09</a></h2>
                <footer class="post-info">
                    <address class="vcard author">By
                        <a class="url fn" href="/author/cao-chau-vu.html">Cao-chau Vu</a>
                    </address>
                    <abbr class="published" title="2015-02-03T00:00:00+01:00">  on 03.02.2015 </abbr>
                </footer><!-- /.post-info -->
	    </header>
                <div class="entry-content"> <p>20150203_153000_lecture_notes</p>
<h1>Notes</h1>
<h2>Least Squares</h2>
<ol>
<li>For each candidate tree <span class="math">\(\tau\)</span>, find branch length that minimizes:<ul>
<li><span class="math">\(\sum_{i &lt; j} (d_{ij}-e_{ij})^2\)</span></li>
<li>Where <span class="math">\(d_{ij}\)</span> is the expected distance calculated from the sum of the branch length and <span class="math">\(e_{ij}\)</span> is the tree distance calculated from distance matrix.</li>
</ul>
</li>
<li>We call ...</li></ol> </div><!-- /.entry-content -->
        </article></li>
        <li><article class="hentry">
            <header class="article-list"> <h2 class="entry-title"><a href="/bio-414-lecture-notes-08.html" rel="bookmark" title="link to Bio 414 Lecture Notes 08">Bio 414 Lecture Notes 08</a></h2>
                <footer class="post-info">
                    <address class="vcard author">By
                        <a class="url fn" href="/author/cao-chau-vu.html">Cao-chau Vu</a>
                    </address>
                    <abbr class="published" title="2015-01-29T00:00:00+01:00">  on 29.01.2015 </abbr>
                </footer><!-- /.post-info -->
	    </header>
                <div class="entry-content"> <p>20150129_153000_lecture_notes</p>
<h1>Notes</h1>
<h2>Distance-Based Phylogenetic Reconstruction Review and Background</h2>
<p><strong>Bird's Eye View</strong></p>
<blockquote>
<p>Sequence Data <span class="math">\(\rightarrow\)</span> Distance Values <span class="math">\(\rightarrow\)</span> Phylogenetic Tree</p>
</blockquote>
<p><strong>Clustering Methods</strong></p>
<ul>
<li>UPGMA</li>
<li>NJ (fast and good)</li>
</ul>
<p><strong>Review of Tree Terminology</strong></p>
<ul>
<li><strong>Tree</strong> Graph without cycles.<ul>
<li>Use to represent phylogenies. </li>
<li>Can be drawn an different way without changing relationships, e.g ...</li></ul></li></ul> </div><!-- /.entry-content -->
        </article></li>
        <li><article class="hentry">
            <header class="article-list"> <h2 class="entry-title"><a href="/bio-414-lecture-notes-07.html" rel="bookmark" title="link to Bio 414 Lecture Notes 07">Bio 414 Lecture Notes 07</a></h2>
                <footer class="post-info">
                    <address class="vcard author">By
                        <a class="url fn" href="/author/cao-chau-vu.html">Cao-chau Vu</a>
                    </address>
                    <abbr class="published" title="2015-01-27T00:00:00+01:00">  on 27.01.2015 </abbr>
                </footer><!-- /.post-info -->
	    </header>
                <div class="entry-content"> <p>20150127_140400_bio_414_lecture_notes</p>
<h1>Notes</h1>
<h2>Substitution Models</h2>
<h3>Jukes-Cantor (JC69)</h3>
<table>
<thead>
<tr>
<th></th>
<th align="center">A</th>
<th align="center">G</th>
<th align="center">C</th>
<th align="center">T</th>
</tr>
</thead>
<tbody>
<tr>
<td>A</td>
<td align="center">-</td>
<td align="center"><span class="math">\(\alpha\)</span></td>
<td align="center"><span class="math">\(\alpha\)</span></td>
<td align="center"><span class="math">\(\alpha\)</span></td>
</tr>
<tr>
<td>G</td>
<td align="center"><span class="math">\(\alpha\)</span></td>
<td align="center">-</td>
<td align="center"><span class="math">\(\alpha\)</span></td>
<td align="center"><span class="math">\(\alpha\)</span></td>
</tr>
<tr>
<td>C</td>
<td align="center"><span class="math">\(\alpha\)</span></td>
<td align="center"><span class="math">\(\alpha\)</span></td>
<td align="center">-</td>
<td align="center"><span class="math">\(\alpha\)</span></td>
</tr>
<tr>
<td>T</td>
<td align="center"><span class="math">\(\alpha\)</span></td>
<td align="center"><span class="math">\(\alpha\)</span></td>
<td align="center"><span class="math">\(\alpha\)</span></td>
<td align="center">-</td>
</tr>
</tbody>
</table>
<p>Stationary/ limiting nucleotide frequencies: </p>
<div class="math">$$\vec{\pi} = \frac{1}{4}, \frac{1}{4}, \frac{1}{4}, \frac{1}{4}$$</div>
<h3>Kimura 2 Parameter Model (K2P)</h3>
<table>
<thead>
<tr>
<th></th>
<th align="center">A ...</th></tr></thead></table> </div><!-- /.entry-content -->
        </article></li>
        <li><article class="hentry">
            <header class="article-list"> <h2 class="entry-title"><a href="/bio-414-lecture-notes-06.html" rel="bookmark" title="link to Bio 414 Lecture Notes 06">Bio 414 Lecture Notes 06</a></h2>
                <footer class="post-info">
                    <address class="vcard author">By
                        <a class="url fn" href="/author/cao-chau-vu.html">Cao-chau Vu</a>
                    </address>
                    <abbr class="published" title="2015-01-22T00:00:00+01:00">  on 22.01.2015 </abbr>
                </footer><!-- /.post-info -->
	    </header>
                <div class="entry-content"> <p>20150122_140400_bio_414_lecture_notes</p>
<h1>Notes</h1>
<h2>Genetics Distances</h2>
<p>Counting polymorphic sites (Hamming distance) doesn't work because we don't observe substitutions. </p>
<p>We can observe changes, but not the process that resulted in that change. For all the below example, assumes one ancestor and two descendants that might have a changed base or not ...</p> </div><!-- /.entry-content -->
        </article></li>
        <li><article class="hentry">
            <header class="article-list"> <h2 class="entry-title"><a href="/bio-414-lecture-notes-05.html" rel="bookmark" title="link to Bio 414 Lecture Notes 05">Bio 414 Lecture Notes 05</a></h2>
                <footer class="post-info">
                    <address class="vcard author">By
                        <a class="url fn" href="/author/cao-chau-vu.html">Cao-chau Vu</a>
                    </address>
                    <abbr class="published" title="2015-01-20T00:00:00+01:00">  on 20.01.2015 </abbr>
                </footer><!-- /.post-info -->
	    </header>
                <div class="entry-content"> <p>20150120_140000_bio_414_lecture_notes</p>
<h1>Notes</h1>
<h2>Blast</h2>
<p><strong>BLAST:</strong> Basic Local Alignment Search Tool</p>
<p><strong>Objective:</strong> Find a match to the query sequence in database.</p>
<p>We don't want to align to all sequences, it's unrealistic with how large databases are. Instead BLAST uses clever heuristics to filter out insignificant results.</p>
<h3>BLAST Output</h3>
<p><strong>Raw score ...</strong></p> </div><!-- /.entry-content -->
        </article></li>
        <li><article class="hentry">
            <header class="article-list"> <h2 class="entry-title"><a href="/bio-414-lecture-notes-04.html" rel="bookmark" title="link to Bio 414 Lecture Notes 04">Bio 414 Lecture Notes 04</a></h2>
                <footer class="post-info">
                    <address class="vcard author">By
                        <a class="url fn" href="/author/cao-chau-vu.html">Cao-chau Vu</a>
                    </address>
                    <abbr class="published" title="2015-01-15T00:00:00+01:00">  on 15.01.2015 </abbr>
                </footer><!-- /.post-info -->
	    </header>
                <div class="entry-content"> <p>20150115_140000_bio_414_lecture_notes</p>
<h1>Notes</h1>
<h2>Sequence Alignments</h2>
<ul>
<li>Used in all evolutionary analysis</li>
<li>Goal of sequence alignment is to group homologous nucleotides/ aa that most likely share a common ancestry.</li>
</ul>
<h2>Good vs. Bad Alignments:</h2>
<p>Take for example the two following sequence</p>
<blockquote>
<p>Alignment 1</p>
<p>CAGT_ATTT</p>
<p>CAGTCAT__</p>
<p>Alignment 2</p>
<p>_CAGTATTT</p>
<p>CA_GTC_AT</p>
</blockquote>
<ul>
<li>The first alignment is good ...</li></ul> </div><!-- /.entry-content -->
        </article></li>
</ul><!-- /#posts-list -->
<p class="paginator">
    Page 1 / 2
        <a href="/index2.html">&raquo;</a>
</p>
</section><!-- /#content -->
      <footer>
	  <a href="/pages/about.html" >About</a>
	  <div class="right-footer">
	  <a href="/categories.html" >Categories</a>
	  <a href="/tags.html" >Tags</a>
	  </div>
      </footer>
</body>
</html>