<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">

<head>

  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
  
  <title>The Commonplace Index - Lecture Notes</title>
<link href="/theme/css/style.css" rel="stylesheet" type="text/css" media="screen" />
<link href="/" type="application/atom+xml" rel="alternate" title="The Commonplace Index ATOM Feed" />
  
  <!--[if IE]>
  <style type="text/css"> 
  .twoColFixRtHdr #mainContent { zoom: 1; }
  </style>
  <![endif]-->

</head>

<body>

<div id="container">

  <div id="header">
  <h1><a href="/index.html">The Commonplace Index</h1>
  
  </div><!-- end #header -->
  
  <div id="topMenu">
  
    <ul>
    	<li><a href="/index.html">Home</a></li>
        <li>&ndash;</li>
        <li><a href="/archives.html">Archives</a></li>
        
    </ul>
  
  </div><!-- end #topMenu -->

  <div id="sidebar">
    <ul>
	<li>
	    <h3>Pages</h3>
	    <ul>
	    </ul>
	</li>
	<li>
            <h3>Categories</h3>
            <ul>
		     <li ><a href="/category/blog.html">Blog</a></li>
		     <li class="active"><a href="/category/lecture-notes.html">Lecture Notes</a></li>
	    </ul>
	</li>
	<li>
		<h3>Blogroll</h3>
		<ul>
                    <li><a href="http://getpelican.com/">Pelican</a></li>
                    <li><a href="http://python.org/">Python.org</a></li>
                    <li><a href="http://jinja.pocoo.org/">Jinja2</a></li>
		</ul>
	</li>
        <li>
                <h3>Social</h3>
                <ul>
                    <li><a href="#">Maybe coming soon</a></li>
                </ul>
        </li><!-- /.social -->
    </ul>
  </div><!-- end #sidebar -->

  <div id="mainContent">
  	
 
        
  	
    <div class="blogItem">
    
      <h2><a href="/bio-414-lecture-notes-12.html">Bio 414 Lecture Notes 12</a></h2>
      <h3>Thu 12 February 2015</h3>
      
      <p>20150212_140000_bio_414_lecture_notes </p>
<h1>Notes</h1>
<h2>MLE Review</h2>
<p>The MLE method find parameter(s) that maximize the probability of observed data. </p>
<p>Let's say we fix the phylogeny and keep only transition/ transversion ratio as free parameter (Kimura model), the maximum likelihood estimation will kind of look like a parabola. Where the maximum point is the MLE transition/ transversion ratio. In example it's about 5. </p>
<p>However, there is a drawback of the need to specify the model correctly. </p>
<h2>Evaluating Phylogenetic Methods</h2>
<p>Since we rarely will know the true phylogeny, we use simulations to evaluate phylogenetic reconstruction algorithms. We can use models of molecular evolution to create this simulation.The model would do many simulations and count how many times the simulation matched our estimated model. </p>
<p>The longer the sequence, the higher the probability that the model will simulate our estimated phylogeny. This doesn't hold true for unweighted parsimony however,  also recall that UPGMA doesn't do well with non-ultrametric trees so if it's a non-ultrametric tree then the probability will also decrease with increasing sequence length. </p>
<h2>Bootstrap</h2>
<h4>Question</h4>
<p>How do we now how much of the estimated phylogeny is due to sampling error?</p>
<p>E.g. Drift, random mutations that aren't relevant, other genetic noise.</p>
<h4>Solution</h4>
<p>We want to "re-run" the evolution using the true phylogeny, but since we don't know the true phylogeny that's technically impossible. What we can do instead is to create two pseudoreplicates. </p>
<p>Example below with trees made by parsimony (though you can have different models used)</p>
<p><strong>Original Alignment</strong></p>
<blockquote>
<p>AATCC</p>
<p>AATTC</p>
<p>ACTCT</p>
<p>ACGTT</p>
</blockquote>
<p>Parsimony Propose Tree: ((1,2),(3,4))</p>
<p><strong>Bootstrap Replicate 1</strong></p>
<blockquote>
<p>TATCC</p>
<p>TATCC</p>
<p>TCTTT</p>
<p>GCGTT</p>
</blockquote>
<p>Parsimony Propose Tree: ((1,2),(3,4))</p>
<p><strong>Bootstrap Replicate 2</strong></p>
<blockquote>
<p>CCCAA</p>
<p>CTTAA</p>
<p>TCCAA</p>
<p>TTTAA</p>
</blockquote>
<p>Parsimony Propose Tree: ((1,3),(2,4))</p>
<p>So here we see bootstrap replicate 1 simulates a similar tree to our original one whereas bootstrap replicate 2 doesn't. If we have enough replicates agree with our original (90%&lt;), then we can say our original tree is most likely right. </p>
<p>Assuming that the alignment sites are independent and follow the same model, bootstraps allow us to prescribe to sample sites with replacement (meaning to resample a site multiple times) to form fake alignments that match the number of sites of the original alignment. </p>
<p>For each replicate, we can use our own chosen phylogenetic method to reconstruct phylogeny (so instead of parsimony above we can use other). One thing to think about is that if we use genetic distance method, we would need to reconstruct the genetic distance each time, so it's computationally expensive. Similarities among phylogenies tells us about the <strong>precision</strong> of our phylogenetic estimation. We have high confidence in parts of the phylogeny that reappear often (have high bootstrap #, &gt;90%). </p>
<p>If our distribution of pseudoreplicates means closely matches our sample mean, then we can have a high confidence interval and can more strongly say our phylogeny closely matches true phylogeny. </p>
<h4>What do we do with these estimates?</h4>
<p>We take our estimated phylogeny (from real data) and "decompose" it into <strong>splits</strong>. We only make splits for internal nodes, not terminal nodes.</p>
<p>For example, with an unrooted tree with two internal nodes each with two terminal nodes (1,2) and (3,4) and one separate terminal node (5), there are two possible splits.</p>
<div class="math">$$((1,2),(3,4,5))\ and\ ((1,2,5),(3,4))$$</div>
<p>We compute splits bootstrap frequency and map them back on the estimated phylogeny. We can have different bootstrap probabilities for each:</p>
<p>((1,2),(3,4,5)) - 90%</p>
<p>((1,2,5),(3,4)) - 60%</p>
<p>What if our estimated phylogeny has only low frequency splits?</p>
<p>Then we can't say we have high confidence in our phylogenetic tree. We want to map uncertainty on to a "central" bootstrap tree. Where <strong>central</strong> usually means <strong>majority consensus tree</strong>. </p>
<ol>
<li>Decompose each tree in a (bootstrap) sample of trees into splits.</li>
<li>Make a list of splits with a frequency of <span class="math">\(\geq 50 \%\)</span>. </li>
<li>Make a new tree with list of splits. That's how you get internal nodes with more than two child nodes. </li>
</ol>
<script type="text/javascript">if (!document.getElementById('mathjaxscript_pelican_#%@#$@#')) {
    var align = "center",
        indent = "0em",
        linebreak = "false";

    if (false) {
        align = (screen.width < 768) ? "left" : align;
        indent = (screen.width < 768) ? "0em" : indent;
        linebreak = (screen.width < 768) ? 'true' : linebreak;
    }
    
    var mathjaxscript = document.createElement('script');
    mathjaxscript.id = 'mathjaxscript_pelican_#%@#$@#';
    mathjaxscript.type = 'text/javascript';
    mathjaxscript.src = '//cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML';
    mathjaxscript[(window.opera ? "innerHTML" : "text")] =
        "MathJax.Hub.Config({" +
        "    config: ['MMLorHTML.js']," +
        "    TeX: { extensions: ['AMSmath.js','AMSsymbols.js','noErrors.js','noUndefined.js'], equationNumbers: { autoNumber: 'AMS' } }," +
        "    jax: ['input/TeX','input/MathML','output/HTML-CSS']," +
        "    extensions: ['tex2jax.js','mml2jax.js','MathMenu.js','MathZoom.js']," +
        "    displayAlign: '"+ align +"'," +
        "    displayIndent: '"+ indent +"'," +
        "    showMathMenu: true," +
        "    tex2jax: { " +
        "        inlineMath: [ ['\\\\(','\\\\)'] ], " +
        "        displayMath: [ ['$$','$$'] ]," +
        "        processEscapes: true," +
        "        preview: 'TeX'," +
        "    }, " +
        "    'HTML-CSS': { " +
        "        styles: { '.MathJax_Display, .MathJax .mo, .MathJax .mi, .MathJax .mn': {color: 'inherit ! important'} }," +
        "        linebreaks: { automatic: "+ linebreak +", width: '90% container' }," +
        "    }, " +
        "}); ";
    (document.body || document.getElementsByTagName('head')[0]).appendChild(mathjaxscript);
}
</script>	

      
      <h3 class="blogMeta">By <a href="#">Cao-chau Vu</a>, Category: <a href="/category/lecture-notes.html">Lecture Notes</a></h3>
    
    </div><!-- end #blogItem -->
	
        <div class="blogItem">
                <h2>Other articles</h2>
        </div>
        
    
    <div class="blogItem">
    
      <h2><a href="/bio-414-lecture-notes-11.html">Bio 414 Lecture Notes 11</a></h2>
      <h3>Tue 10 February 2015</h3>
      
      <p>20150210_140000_bio_414_lecture_notes_11</p>
<h1>Notes</h1>
<p>Today is all about <strong>maximum likelihood phylogenetics</strong>. </p>
<h2>Maximum Likelihood Phylogenetics Examples</h2>
<h3>A 2 Taxa Case</h3>
<p>If we have one ancestor (X, for having unknown base X) with two descendants (A,G) with corresponding time (<span class="math">\(t_{1}, t_{2}\)</span>), that is: </p>
<div class="math">$$(A: t_{1},G:t_{2})X$$</div>
<p>The ...</p>	
<h3 class="blogMeta">By <a href="#">Cao-chau Vu</a>, Category: <a href="/category/lecture-notes.html">Lecture Notes</a></h3>
	<a href="/bio-414-lecture-notes-11.html">Continue reading …</a>
      
      
	
    
    </div><!-- end #blogItem -->
        
    
    <div class="blogItem">
    
      <h2><a href="/bio-414-lecture-notes-10.html">Bio 414 Lecture Notes 10</a></h2>
      <h3>Thu 05 February 2015</h3>
      
      <p>20150205_140000_bio_414_lecture_notes </p>
<h1>Notes</h1>
<h2>Parsimony Continued</h2>
<h3>Generalized Parsimony</h3>
<p>We added 1 for every substitution required to explain site data, this corresponds to a (step) matrix like: </p>
<table>
<thead>
<tr>
<th></th>
<th>A</th>
<th>G</th>
<th>C</th>
<th>T</th>
</tr>
</thead>
<tbody>
<tr>
<td>A</td>
<td>0</td>
<td>1</td>
<td>1</td>
<td>1</td>
</tr>
<tr>
<td>G</td>
<td>1</td>
<td>0</td>
<td>1</td>
<td>1</td>
</tr>
<tr>
<td>C</td>
<td>1</td>
<td>1</td>
<td>0</td>
<td>1</td>
</tr>
<tr>
<td>T</td>
<td>1</td>
<td>1</td>
<td>1</td>
<td>0</td>
</tr>
</tbody>
</table>
<p>What if ...</p>	
<h3 class="blogMeta">By <a href="#">Cao-chau Vu</a>, Category: <a href="/category/lecture-notes.html">Lecture Notes</a></h3>
	<a href="/bio-414-lecture-notes-10.html">Continue reading …</a>
      
      
	
    
    </div><!-- end #blogItem -->
        
    
    <div class="blogItem">
    
      <h2><a href="/bio-414-lecture-notes-09.html">Bio 414 Lecture Notes 09</a></h2>
      <h3>Tue 03 February 2015</h3>
      
      <p>20150203_153000_lecture_notes</p>
<h1>Notes</h1>
<h2>Least Squares</h2>
<ol>
<li>For each candidate tree <span class="math">\(\tau\)</span>, find branch length that minimizes:<ul>
<li><span class="math">\(\sum_{i &lt; j} (d_{ij}-e_{ij})^2\)</span></li>
<li>Where <span class="math">\(d_{ij}\)</span> is the expected distance calculated from the sum of the branch length and <span class="math">\(e_{ij}\)</span> is the tree distance calculated from distance matrix.</li>
</ul>
</li>
<li>We call ...</li></ol>	
<h3 class="blogMeta">By <a href="#">Cao-chau Vu</a>, Category: <a href="/category/lecture-notes.html">Lecture Notes</a></h3>
	<a href="/bio-414-lecture-notes-09.html">Continue reading …</a>
      
      
	
    
    </div><!-- end #blogItem -->
        
    
    <div class="blogItem">
    
      <h2><a href="/bio-414-lecture-notes-08.html">Bio 414 Lecture Notes 08</a></h2>
      <h3>Thu 29 January 2015</h3>
      
      <p>20150129_153000_lecture_notes</p>
<h1>Notes</h1>
<h2>Distance-Based Phylogenetic Reconstruction Review and Background</h2>
<p><strong>Bird's Eye View</strong></p>
<blockquote>
<p>Sequence Data <span class="math">\(\rightarrow\)</span> Distance Values <span class="math">\(\rightarrow\)</span> Phylogenetic Tree</p>
</blockquote>
<p><strong>Clustering Methods</strong></p>
<ul>
<li>UPGMA</li>
<li>NJ (fast and good)</li>
</ul>
<p><strong>Review of Tree Terminology</strong></p>
<ul>
<li><strong>Tree</strong> Graph without cycles.<ul>
<li>Use to represent phylogenies. </li>
<li>Can be drawn an different way without changing relationships, e.g ...</li></ul></li></ul>	
<h3 class="blogMeta">By <a href="#">Cao-chau Vu</a>, Category: <a href="/category/lecture-notes.html">Lecture Notes</a></h3>
	<a href="/bio-414-lecture-notes-08.html">Continue reading …</a>
      
      
	
    
    </div><!-- end #blogItem -->
        
    
    <div class="blogItem">
    
      <h2><a href="/bio-414-lecture-notes-07.html">Bio 414 Lecture Notes 07</a></h2>
      <h3>Tue 27 January 2015</h3>
      
      <p>20150127_140400_bio_414_lecture_notes</p>
<h1>Notes</h1>
<h2>Substitution Models</h2>
<h3>Jukes-Cantor (JC69)</h3>
<table>
<thead>
<tr>
<th></th>
<th align="center">A</th>
<th align="center">G</th>
<th align="center">C</th>
<th align="center">T</th>
</tr>
</thead>
<tbody>
<tr>
<td>A</td>
<td align="center">-</td>
<td align="center"><span class="math">\(\alpha\)</span></td>
<td align="center"><span class="math">\(\alpha\)</span></td>
<td align="center"><span class="math">\(\alpha\)</span></td>
</tr>
<tr>
<td>G</td>
<td align="center"><span class="math">\(\alpha\)</span></td>
<td align="center">-</td>
<td align="center"><span class="math">\(\alpha\)</span></td>
<td align="center"><span class="math">\(\alpha\)</span></td>
</tr>
<tr>
<td>C</td>
<td align="center"><span class="math">\(\alpha\)</span></td>
<td align="center"><span class="math">\(\alpha\)</span></td>
<td align="center">-</td>
<td align="center"><span class="math">\(\alpha\)</span></td>
</tr>
<tr>
<td>T</td>
<td align="center"><span class="math">\(\alpha\)</span></td>
<td align="center"><span class="math">\(\alpha\)</span></td>
<td align="center"><span class="math">\(\alpha\)</span></td>
<td align="center">-</td>
</tr>
</tbody>
</table>
<p>Stationary/ limiting nucleotide frequencies: </p>
<div class="math">$$\vec{\pi} = \frac{1}{4}, \frac{1}{4}, \frac{1}{4}, \frac{1}{4}$$</div>
<h3>Kimura 2 Parameter Model (K2P)</h3>
<table>
<thead>
<tr>
<th></th>
<th align="center">A ...</th></tr></thead></table>	
<h3 class="blogMeta">By <a href="#">Cao-chau Vu</a>, Category: <a href="/category/lecture-notes.html">Lecture Notes</a></h3>
	<a href="/bio-414-lecture-notes-07.html">Continue reading …</a>
      
      
	
    
    </div><!-- end #blogItem -->
        
    
    <div class="blogItem">
    
      <h2><a href="/bio-414-lecture-notes-06.html">Bio 414 Lecture Notes 06</a></h2>
      <h3>Thu 22 January 2015</h3>
      
      <p>20150122_140400_bio_414_lecture_notes</p>
<h1>Notes</h1>
<h2>Genetics Distances</h2>
<p>Counting polymorphic sites (Hamming distance) doesn't work because we don't observe substitutions. </p>
<p>We can observe changes, but not the process that resulted in that change. For all the below example, assumes one ancestor and two descendants that might have a changed base or not ...</p>	
<h3 class="blogMeta">By <a href="#">Cao-chau Vu</a>, Category: <a href="/category/lecture-notes.html">Lecture Notes</a></h3>
	<a href="/bio-414-lecture-notes-06.html">Continue reading …</a>
      
      
	
    
    </div><!-- end #blogItem -->
        
    
    <div class="blogItem">
    
      <h2><a href="/bio-414-lecture-notes-05.html">Bio 414 Lecture Notes 05</a></h2>
      <h3>Tue 20 January 2015</h3>
      
      <p>20150120_140000_bio_414_lecture_notes</p>
<h1>Notes</h1>
<h2>Blast</h2>
<p><strong>BLAST:</strong> Basic Local Alignment Search Tool</p>
<p><strong>Objective:</strong> Find a match to the query sequence in database.</p>
<p>We don't want to align to all sequences, it's unrealistic with how large databases are. Instead BLAST uses clever heuristics to filter out insignificant results.</p>
<h3>BLAST Output</h3>
<p><strong>Raw score ...</strong></p>	
<h3 class="blogMeta">By <a href="#">Cao-chau Vu</a>, Category: <a href="/category/lecture-notes.html">Lecture Notes</a></h3>
	<a href="/bio-414-lecture-notes-05.html">Continue reading …</a>
      
      
	
    
    </div><!-- end #blogItem -->
        
    
    <div class="blogItem">
    
      <h2><a href="/bio-414-lecture-notes-04.html">Bio 414 Lecture Notes 04</a></h2>
      <h3>Thu 15 January 2015</h3>
      
      <p>20150115_140000_bio_414_lecture_notes</p>
<h1>Notes</h1>
<h2>Sequence Alignments</h2>
<ul>
<li>Used in all evolutionary analysis</li>
<li>Goal of sequence alignment is to group homologous nucleotides/ aa that most likely share a common ancestry.</li>
</ul>
<h2>Good vs. Bad Alignments:</h2>
<p>Take for example the two following sequence</p>
<blockquote>
<p>Alignment 1</p>
<p>CAGT_ATTT</p>
<p>CAGTCAT__</p>
<p>Alignment 2</p>
<p>_CAGTATTT</p>
<p>CA_GTC_AT</p>
</blockquote>
<ul>
<li>The first alignment is good ...</li></ul>	
<h3 class="blogMeta">By <a href="#">Cao-chau Vu</a>, Category: <a href="/category/lecture-notes.html">Lecture Notes</a></h3>
	<a href="/bio-414-lecture-notes-04.html">Continue reading …</a>
      
      
	
    
    </div><!-- end #blogItem -->
        
    
    <div class="blogItem">
    
      <h2><a href="/bio-414-lecture-notes-03.html">Bio 414 Lecture Notes 03</a></h2>
      <h3>Tue 13 January 2015</h3>
      
      <p>20150113_140000_bio_414_lecture_notes</p>
<h1>Notes</h1>
<h2>Mutations Continued</h2>
<ul>
<li><strong>Mutation:</strong> any heritable change to the original genome.</li>
<li><strong>Point mutation:</strong> Replacement of single nucleotide by another one.<ul>
<li>Two class of point mutation.</li>
<li><strong>Transition:</strong> same class, e.g. purine-&gt;purine.</li>
<li><strong>Tranversion:</strong> different class, e.g. purine-&gt;pyrimidine. </li>
</ul>
</li>
<li><strong>Insertions and deletions (indel)</strong><ul>
<li>Can cause frame shifts, usually resulting ...</li></ul></li></ul>	
<h3 class="blogMeta">By <a href="#">Cao-chau Vu</a>, Category: <a href="/category/lecture-notes.html">Lecture Notes</a></h3>
	<a href="/bio-414-lecture-notes-03.html">Continue reading …</a>
      
      
	
    
    </div><!-- end #blogItem -->
        
    
    <div class="blogItem">
    
      <h2><a href="/bio-414-lecture-notes-02.html">Bio 414 Lecture Notes 02</a></h2>
      <h3>Thu 08 January 2015</h3>
      
      <p>20150108_140000_bio_414_lecture_notes</p>
<h1>Notes</h1>
<h2>Homework</h2>
<ul>
<li>.rmd file</li>
<li>On website</li>
<li>Due next thursday</li>
<li>Upload as .rmd file</li>
</ul>
<h2>Sequencing</h2>
<h3>Sanger</h3>
<ul>
<li>A series of starts and stops.</li>
<li>Break DNA into smaller "reads" (500-600)</li>
<li>In a tube mix:<ul>
<li>Template sequence: The thing you want to sequence.</li>
<li>Primer: A short known sequence complementary to portion of template ...</li></ul></li></ul>	
<h3 class="blogMeta">By <a href="#">Cao-chau Vu</a>, Category: <a href="/category/lecture-notes.html">Lecture Notes</a></h3>
	<a href="/bio-414-lecture-notes-02.html">Continue reading …</a>
      
      
	
    
    </div><!-- end #blogItem -->
        
    
    <div class="blogItem">
    
      <h2><a href="/bio-414-lecture-notes-01.html">Bio 414 Lecture Notes 01</a></h2>
      <h3>Tue 06 January 2015</h3>
      
      <h1>Notes</h1>
<p>Why study molecular evolution?</p>
<ul>
<li>To build a tree of life.</li>
</ul>
<p>Why are phylogenetics trees important?</p>
<ul>
<li>Formulate hypothesis. </li>
<li>Compare organisms. </li>
<li>Geographic and time spread.</li>
<li><strong>To understand (human) evolution.</strong></li>
</ul>
<p>Why is it useful to study human evolution?</p>
<ul>
<li>Incorporate into other analysis, if we know which genes are related to geographical origin ...</li></ul>	
<h3 class="blogMeta">By <a href="#">Cao-chau Vu</a>, Category: <a href="/category/lecture-notes.html">Lecture Notes</a></h3>
	<a href="/bio-414-lecture-notes-01.html">Continue reading …</a>
      
      
	
    
    </div><!-- end #blogItem -->
  
  </div><!-- end #mainContent -->

<div class="clearfloat"></div>

  <div id="footer">
  
    <p><a href="http://css4free.com/" title="free CSS web site designs">Free CSS Gallery</a></p>
    <p>Proudly powered by <a href="http://getpelican.com">pelican</a>, which takes great advantages of <a href="http://python.org">python</a>.
  </div><!-- end #footer -->

</div><!-- end #container -->
</body>
</html>